packages/ui/src/ui/utils/scheduling/pool-child.js

```diff
@@ -44,124 +44,156 @@ function preparePoolChildResource(data, type, treeResources, resource) {
-    data.accumulated = ypath.getValue(attributes, '/accumulated_resource_ratio_volume');
-    data.accumulatedCpu = ypath.getValue(attributes, '/accumulated_resource_volume/cpu');
-    data.burstDuration = ypath.getValue(attributes, '/estimated_burst_usage_duration_sec');
-
-    const fifoSortParams = _.map(
-        ypath.getValue(cypressAttributes, '/fifo_sort_parameters'),
-        (param) => ypath.getValue(param),
-    );
-    data.fifoSortParams =
-        fifoSortParams.length > 0 ? fifoSortParams : ['start_time', 'weight', 'pending_job_count'];
-    data.abc = ypath.getValue(cypressAttributes, '/abc') || {};
-    data.acl = ypath.getValue(cypressAttributes, '/acl') || [];
-    data.forbidImmediateOperations =
-        ypath.getBoolean(cypressAttributes, '/forbid_immediate_operations') || false;
-    data.createEphemeralSubpools =
-        ypath.getBoolean(cypressAttributes, '/create_ephemeral_subpools') || false;
-
-    // Resources
-    data.dominantResource = ypath.getValue(attributes, '/dominant_resource');
-
-    data.resources = {};
-
-    preparePoolChildResource(data, type, treeResources, 'cpu');
-    preparePoolChildResource(data, type, treeResources, 'user_memory');
-    preparePoolChildResource(data, type, treeResources, 'gpu');
-    preparePoolChildResource(data, type, treeResources, 'user_slots');
+        data.accumulated = ypath.getValue(attributes, '/accumulated_resource_ratio_volume');
+        data.accumulatedCpu = ypath.getValue(attributes, '/accumulated_resource_volume/cpu');
+        data.burstDuration = ypath.getValue(attributes, '/estimated_burst_usage_duration_sec');
 
-    return data;
+        const fifoSortParams = _.map(
+            ypath.getValue(cypressAttributes, '/fifo_sort_parameters'),
+            (param) => ypath.getValue(param),
+        );
+        data.fifoSortParams =
+            fifoSortParams.length > 0
+                ? fifoSortParams
+                : ['start_time', 'weight', 'pending_job_count'];
+        data.abc = ypath.getValue(cypressAttributes, '/abc') || {};
+        data.acl = ypath.getValue(cypressAttributes, '/acl') || [];
+        data.forbidImmediateOperations =
+            ypath.getBoolean(cypressAttributes, '/forbid_immediate_operations') || false;
+        data.createEphemeralSubpools =
+            ypath.getBoolean(cypressAttributes, '/create_ephemeral_subpools') || false;
+
+        // Resources
+        data.dominantResource = ypath.getValue(attributes, '/dominant_resource');
+
+        data.resources = {};
+
+        preparePoolChildResource(data, type, treeResources, 'cpu');
+        preparePoolChildResource(data, type, treeResources, 'user_memory');
+        preparePoolChildResource(data, type, treeResources, 'gpu');
+        preparePoolChildResource(data, type, treeResources, 'user_slots');
+
+        return data;
+    } catch (e) {
+        const error = {
+            message: `An error occured while parsing pool "${data.name}" data:`,
+            inner_errors: [e],
```

Comment:
The same as above. What about adding `inner_errors` instead of wrapping by an object?